_includes/scripts.html

```diff
@@ -3,3 +3,4 @@
   <script src="{{ site.baseurl }}/assets/js/vendor/respond.js"></script>
   <script src="{{ site.baseurl }}/assets/js/vendor/rem.min.js"></script>
   <script src="{{ site.baseurl }}/assets/js/main.js"></script>
+  <script src="{{ site.baseurl }}/assets/js/vendor/prism.js"></script>
```

Comment:
Vendor libs should go before our own, so this should be moved to the line before main.js